CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -696,50 +701,6 @@ static void __InitTZStrings(void) {
     }
     __CFUnlock(&__CFTZDirLock);
 }
-
-#elif DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED
```

Comment:
We sure it's correct to remove this?

Reply:
Tests pass, and it matches the original.